pbrd/pbr_zebra.c

```diff
@@ -549,7 +549,14 @@ static void pbr_encode_pbr_map_sequence(struct stream *s,
 		stream_putl(s, pbr_nht_get_table(pbrms->nhgrp_name));
 	else if (pbrms->nhg)
 		stream_putl(s, pbr_nht_get_table(pbrms->internal_nhg_name));
+	else {
+		/* Not valid for install without table */
```

Comment:
If this condition arises a rule installed without table ,  can you please add a debug message in this?

Reply:
nvm, i see its there in caller function :)